core/lib/basic_types/src/web3/mod.rs

```diff
@@ -354,8 +354,8 @@ pub struct BlockHeader {
     #[serde(rename = "gasLimit", default)]
     pub gas_limit: U256,
     /// Base fee per unit of gas (if past London)
-    #[serde(rename = "baseFeePerGas", skip_serializing_if = "Option::is_none")]
-    pub base_fee_per_gas: Option<U256>,
```

Comment:
Era started to worked *after* London fork so `base_fee_per_gas` should be present.

Reply:
Types in this file are used by L1 eth client. As you've stated all era networks (testnet, mainnet) started to work after London hard fork, but I would still leave `Option` to make it possible to query pre-London blocks -- can be required for some cases, I don't think we should limit eth client functionality.